### PR TITLE
NPE for status_details

### PR DESCRIPTION
**Pull request by dmitchell**
https://github.com/googleads/google-ads-ruby/issues/22

### CODE REVIEW

File: lib/google/ads/google_ads/logging_interceptor.rb [hunk 1]
```diff
@@ -52,7 +52,7 @@ def request_response(request:, call:, method:, metadata: {})
             response_message = "Incoming response (errors): \n"
 
             most_recent_error = Google::Gax::GaxError.new('')
-            most_recent_error.status_details.each do |detail|
+            most_recent_error.status_details&.each do |detail|
```
Comment (mcloonan) [inline lib/google/ads/google_ads/logging_interceptor.rb]: The &. operator was added in Ruby 2.3. We'd prefer not to restrict support to earlier versions unnecessarily, even if they are rarely used. Can you implement this a different way?

**Comment by dmitchell** [comment]
Done but github won't let me comment on the PR

**Comment by mcloonan** [comment]
Looks like a lot more changes made it in this time somehow. Can you limit it to just the NPE fix? Thanks!

**Comment by dmitchell** [comment]
Yeah, my ide corrected interpolated constant strings to non-interpolated
.... Reverted

**Pull request merged**